docs/usage-and-billing.mdx

```diff
@@ -0,0 +1,158 @@
+
+**Bots** and similar automations are excluded from this count.
+
+### Exceeding the free usage limit
+
+Semgrep scans stop when the usage limit is exceeded. You can resume scanning through the following:
+
+* A one-time 30-day free trial that starts automatically when the usage limit is exceeded for the first time.
+* By purchasing additional licenses.
+* By waiting for the next billing cycle.
+
+If the very first scan exceeds the usage limit, Semgrep still completes the first scan and a one-time 30-day free trial automatically starts. After the trial concludes, if scans are run on private repositories that exceed the usage limit, scans will not run until additional licenses are purchased.
+
+### Usage limit on public projects
+
+Public projects have no limits on contributors.
+
+### Semgrep CLI commands subject to usage limits
+
+The `semgrep scan` command is subject to the usage limit only if the scan is by a logged-in or authenticated contributor. Semgrep computes contributor counts for any logged-in scan command (for example, `semgrep ci`, `semgrep scan`, and so on) when the Pro Engine, Supply Chain, or Pro rules are used.
+
+## Determine your plan needs
+
+Within your team or organization, assess the number of **contributors**. Contributors are members of your organization that make commits. That determines the number of **licenses** needed for the plan purchase.
+
+For example, if a project has 4 unique contributors who create commits during the billing period while Semgrep is scanning their repositories, only 4 licenses are required even if the organization has a total of 10 members. If these unique contributors commit to many projects within the same organization, they are counted once, so no additional cost is charged.
+
+:::info
+You only need to buy licenses for contributors over 10. The 11th contributor and onwards are charged. For example, an engineering team with 20 contributing developers will only pay for 10 licenses.
+:::
+
+All members of the organization, regardless of contributor (license) status, have access to paid features for the chosen tier. This means that project managers and other non-programming roles can still view the Semgrep Cloud Platform dashboard.
+
+### Single-product purchases
+
+Yes, you can buy a single product. Products can also be disabled from the [Settings page](https://semgrep.dev/orgs/-/settings).
+
+### Different number of licenses per product
+
+You must purchase an **equal number of licenses** for each product you intend to use. For example, you cannot purchase 4 licenses of Semgrep Supply Chain and 9 licenses of Semgrep Code. 
+
+## Semgrep add-on reconciliation of licenses
+
+If the organization exceeds the number of purchased licenses, the organization will be charged based on the number of licenses that exceeded the purchased amount. The additional charge starts the month after the use of licenses exceeds the contracted amount.
```

Comment:
I'm struggling to give a clearer alternative here but this does feel confusing. I think the thing that feels odd is that we're saying "you'll be charged based on how much you exceed", but it's not ultra clear why the excess and the original would be separate unless you get into details of the payment structure, which only happens after this

Reply:
I see what you mean. Tried to make it clear that the licenses are bought for a set period of time indicated in a contract.